drivers/hmis/app/models/hmis/ce.rb

```diff
@@ -0,0 +1,10 @@
+module Hmis::Ce
+  def self.table_name_prefix
+    'ce_'
+  end
+
+  def self.enabled?
+    # perhaps this should be an env or db config flag
+    Rails.env.development? || Rails.env.test?
+  end
```

Comment:
I would like to see this as a database config.  It should probably be stored on the data source (or related to the data source.)
I also think we should rework `GrdaWarehouse::Config` to be more of a key value store at some point, not sure we want to tackle that change now.